qiskit_addon_cutting/utils/observable_grouping.py

```diff
@@ -250,13 +250,13 @@ def groups(self) -> list[CommutingObservableGroup]:
 
     @property
     def lookup(self) -> dict[Pauli, list[tuple[int, int]]]:
-        r"""Get dict which maps each :class:`~qiskit.quantum_info.Pauli` observable to a list of indices, ``(i, j)``, to commuting observables in ``groups``.
+        """Get dict which maps each :class:`~qiskit.quantum_info.Pauli` observable to a list of indices, ``(i, j)``, to commuting observables in ``groups``.
 
         For each element of the list, it means that the :class:`~qiskit.quantum_info.Pauli` is given by
-        the ``j``th commuting observable in the ``i``th group.
+        the ``j``-th commuting observable in the ``i``-th group.
 
         This list will be of length 1 at minimum, but may potentially be longer
-        if multiple :class:`.CommutingObservableGroup`\ s are compatible with the given
+        if multiple :class:`.CommutingObservableGroup` objects are compatible with the given
```

Comment:
Best practice is to pluralize like this

Reply:
I don't agree with this change.  Can you provide some more justification than "best practice"?

If I read it aloud, the phrase "multiple commuting observable groups are compatible with the given Pauli" makes sense to me as a physicist (other than Qiskit's weird convention of using "group" rather than "set").  I am not sure what value "objects" brings.

Reply:
The `\s` style with code before it is hard to read and especially hard to globalize/translate. Abby C and Becky were concerned about it. So, the alternative is to use the format "blah blah `MyClass` objects/intances"

> I am not sure what value "objects" brings.

Here, objects == instances of the class. Is there an alternative word you prefer like "instances"?

> hard to globalize/translate

Fwiw, we haven't finalized yet whether we will globalize API docs. Still, it's preferable to use a consistent style between API docs and guides

This isn't a hard requirement - you can push back